src/EventHandler/CallbackQuery.php

```diff
@@ -0,0 +1,122 @@
+<?php declare(strict_types=1);
+
+namespace danog\MadelineProto\EventHandler;
+
+use danog\MadelineProto\MTProto;
+use danog\MadelineProto\ParseMode;
+
+final class CallbackQuery extends Update
```

Comment:
IMO It's a good idea to rename this to AbstractButtonQuery, and create`AbstractChatButtonQuery` and `AbstractInlineButtonQuery` classes, mostly empty except for a `$chatId`+`$messageId` in AbstractChatButtonQuery and a private $messageId in `AbstractInlineButtonQuery` (we shouldn't expose it, it only contains internal info used by the answer method).

Then four final classes specialize the Abstract(Inline)ButtonQuery to `(Inline)(Game|Callback)Query`, where the game and callback have either a `$data` or a `$gameShortName` property.